includes/class-gp-translation-helpers.php

```diff
@@ -64,10 +72,15 @@ public function __construct() {
 		add_action( 'wp_ajax_comment_with_feedback', array( $this, 'comment_with_feedback' ) );
 		add_action( 'wp_ajax_optout_discussion_notifications', array( $this, 'optout_discussion_notifications' ) );
 
+		if ( ! function_exists( 'get_plugins' ) ) {
+			require_once ABSPATH . 'wp-admin/includes/plugin.php';
+		}
+		self::$plugin_version = get_plugins()['gp-translation-helpers/gp-translation-helpers.php']['Version'];
```

Comment:
We should add some fallback and `isset()` check in case the plugin is installed in another directory or similar.

Reply:
- Can `get_plugins()` be called and did it generate decent output (`$plugins = get_plugins(); if ( ! is_array( $plugins ) )`?
- Does it have the the array required entry?
- What to use as version if it doesn't exist.